scripts/swtpm_sim.test

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+#
+# Run tests against SWTPM simulator
+# Setting TESTS to a list of tests to run
+#   TESTS=./examples/native/native_test\ ./examples/wrap/wrap_test ./scripts/swtpm_sim.test
+#
+
+# set -x
+
+TOP_DIR=$(realpath $(dirname $0)/..)
+
+die() {
+    echo $* >&2
+    exit 1
+}
+
+build() {
+    if [ ! -d "$SWTPM_DIR" ]; then
+        echo "Cloning SWTPM from $SWTPM_GIT_URL to $SWTPM_DIR";
+        git clone $SWTPM_GIT_URL $SWTPM_DIR || die "unable to clone $SWTPM_GIT_URL for simualtor";
+    fi
+    echo "Building SWTPM: $(git -C "$SWTPM_DIR" describe --long)";
+    make -C $SWTPM_DIR/src > /dev/null || die "Failed to build $SWTPM_DIR";
+}
+
+TESTS=${TESTS:="${TOP_DIR}/tests/unit.test"}
+SWTPM_GIT_URL=${SWTPM_GIT_URL:="https://github.com/kgoldman/ibmswtpm2.git"}
+
+SWTPM_DIR="$(realpath $(basename $SWTPM_GIT_URL | sed 's/\.git//'))"
+
+# check if on path or clone and build
+which ${SWTPM_EXEC:=tpm_server} > /dev/null
+if [ 0 -ne $? ]; then
+    build 
+    SWTPM_EXEC=$SWTPM_DIR/src/$SWTPM_EXEC
+fi;
+
+# start swtpm server
+$SWTPM_EXEC >/dev/null &
```

Comment:
It might be good to check the return code on this... Although it was nice because I was able to start `tpm_server` myself and run the script and the error was ignored.

Reply:
Adding a warning if it exits immediately. Not sure if there is a better way to show return code.